_DeploymentAndDistroPackaging/ansible/roles/ciao-controller/tasks/certificates.yml

```diff
@@ -43,6 +43,8 @@
         creates: cert-CNCIAgent-localhost.pem
       - role: controller
         creates: cert-Controller-localhost.pem
+      - role: agent,netagent
+        creates: cert-CNAgent-NetworkingAgent-localhost.pem
```

Comment:
This is bad from a security perspective.  It creates a broader attack surface for malicious cluster membership.  It should only be created if it is going to be used.